aspnetcore/host-and-deploy/linux-apache.md

```diff
@@ -200,27 +201,32 @@ Environment=ASPNETCORE_ENVIRONMENT=Production
 WantedBy=multi-user.target
 ```
 
-> [!NOTE]
-> **User** &mdash; If the user *apache* isn't used by the configuration, the user must be created first and given proper ownership for files.
+If the user *apache* isn't used by the configuration, the user must be created first and given proper ownership of files.
 
-> [!NOTE]
-> Some values (for example, SQL connection strings) must be escaped for the configuration providers to read the environment variables. Use the following command to generate a properly escaped value for use in the configuration file:
->
-> ```console
-> systemd-escape "<value-to-escape>"
-> ```
+Use `TimeoutStopSec` to configure the duration of time to wait for the app to shut down after it receives the initial interrupt signal. If the app doesn't shut down in this period, SIGKILL is issued to terminate the app. Provide the value as unitless seconds (for example, `150`), a time span value (for example, `2min 30s`), or `infinity` to disable the timeout. `TimeoutStopSec` defaults to the value of `DefaultTimeoutStopSec` in the manager configuration file (*systemd-system.conf*, *system.conf.d*, *systemd-user.conf*, *user.conf.d*). The default timeout for most distros is 90 seconds.
```

Comment:
Distros is slang. Let's spell it out as "distributions".

Reply:
I have a few quick observations before we go with that change ... let's mull it over for a sec. I have a concern (point 2 below). "Distribution" vs. "distro" is a great example of what I'm concerned about (if my point 1 is correct).

1. "Distro" is used frequently in and out of Linux circles. I've heard plenty of non-Linux devs who know a little bit about Linux use it and seem to understand what it means. Along with "distribution," I see it in written docs around the Net. "Distro" is clearly the informal form of the two words, but I don't think it can be regarded as *slang*. Slang words are distinguished by one key factor: They're understood and used by a subset of people. Example: "I'm gonna *bank* him!" ... only a subset of people (city cats) know that "bank" means "hit" ... it's clearly slang in that context. "Distro" just feels like an informal version of "distribution." If it's regarded as an informal word and not slang, then we can use it. I personally like "distro" over "distribution." It's cool. 😎 I'm curious what @shirhatti thinks: *informal and slang* or merely *informal*?
1. I've mentioned my concern about docs becoming *too formal* in the past. If that's the order, I can certainly deliver completely formal docs. No worries there ... I did that for many years in bio research and education. If we're going formal, then let's go with "distribution." If we'll maintain some informal words and phrases (and my point 1 is correct), then I recommend "distro" here.

Thoughts? cc: @Rick-Anderson @tdykstra @mairaw

Reply:
I vote for "distribution".  I think we should be consistent in our terminology, not using distribution here and distro there. And if we're going to choose one, I wouldn't choose the one that sounds like slang.  But I can see your points as well, so I'd be just as happy if I got outvoted on this one. 😄

Reply:
@tdykstra I agree and don't care about a single word really. I do prefer to write informally tho (if it's permitted). I think there's been a trend in favor of formal writing over the last two years. I just have more *FuN!* 🎈 with a topic if I can *let my hair down with the reader* at least a little bit. Perhaps, it's due to all those years in bio, where I was writing so much up-tight content. :smile:

Reply:
I personally use the word "distros", but I was concerned about the impact on localization.

Reply:
I've asked this to the cloud style team a while ago and they added a specific entry for this in our style guide. It's distribution, not distros.

Reply:
Thanks @mairaw ... I guessed right. I merged it with "distribution." :+1: